src/main/java/com/parkhomovsky/bookstore/controller/BookController.java

```diff
@@ -21,38 +24,45 @@
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/books")
+@Tag(name = "Book management", description = "Endpoints for managing products")
 public class BookController {
     private final BookService bookService;
 
     @PutMapping("/{id}")
     public BookDto update(
-            @PathVariable Long id, @RequestBody CreateBookRequestDto createBookRequestDto) {
+            @PathVariable Long id, @RequestBody @Valid CreateBookRequestDto createBookRequestDto) {
         return bookService.update(id, createBookRequestDto);
     }
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @Operation(summary = "Create new book", description = "Adding new book to database")
     public BookDto createBook(@RequestBody @Valid CreateBookRequestDto book) {
         return bookService.create(book);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Get book by id", description = "Try to find book with provided id")
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.getBookById(id);
     }
 
     @GetMapping("/search")
+    @Operation(summary = "Search books", description = "Search books by title or author")
     public List<BookDto> search(BookSearchParameters parameters) {
         return bookService.bookSearch(parameters);
     }
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
+    @Operation(summary = "List book using pagination", description =
+            "List books on page using parameters")
+    public List<BookDto> findAll(Pageable pageable) {
+        return bookService.getAll(pageable);
     }
 
     @DeleteMapping("/{id}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
+    @Operation(summary = "Delete book by id", description = "Delete book from database")
```

Comment:
Wrong description, you don't delete book from database (remember about soft delete)